molecule/quarkus/prepare.yml

```diff
@@ -1,19 +1,25 @@
 ---
 - name: Prepare
   hosts: all
+  gather_facts: True
   tasks:
     - name: Install sudo
       ansible.builtin.yum:
         name: sudo
         state: present
     - command: openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256 -days 365 -nodes -subj '/CN=instance'
       delegate_to: localhost
+
     - lineinfile:
         dest: /etc/hosts
         line: "127.0.0.1 instance"
         state: present
       delegate_to: localhost
       become: yes
+      when:
```

Comment:
@guidograzioli not the nicest way to figure out when we run on github or olympus, but that works and will do for now if you can leav with it :)